packages/frontend-2/components/workspace/header/Header.vue

```diff
@@ -66,15 +66,18 @@
           </button>
         </div>
         <div class="flex items-center gap-x-3">
-          <button
-            class="block"
-            @click="openSettingsDialog(SettingMenuKeys.Workspace.Members)"
-          >
-            <UserAvatarGroup
-              :users="team.map((teamMember) => teamMember.user)"
-              class="max-w-[104px]"
-            />
-          </button>
+          <div v-tippy="isWorkspaceAdmin ? 'Manage members' : 'View members'">
```

Comment:
Will this work correctly if you switch between workspaces? Sometimes to v-tippy reactive is a bit off

Reply:
Yeah, I tested this expecting that bug, but for some reason it worked fine in this scenario